packages/spotlight/SpotlightRootDecorator/SpotlightRootDecorator.js

```diff
@@ -110,6 +110,10 @@ const SpotlightRootDecorator = hoc(defaultConfig, (config, Wrapped) => {
 				Spotlight.focus();
 			}
 
+			if (this.containerRef === null && typeof document === 'object') {
+				this.containerRef = document.querySelector('#root');
```

Comment:
'#root' is hardcoded here. I prefer adding a new API of SpotlightRootDecorator for "ThemeDecorator"s of theme libraries to let SpotlightRootDecorator know the root node ID.